### PR TITLE
Adds example for itemstack tesla capabilities

### PR DESCRIPTION
**Pull request by adrwa**
Adds two examples for battery that holds 10000 Tesla and a creative battery which produces infinite.

Example for #14

### CODE REVIEW

**Comment by adrwa** [comment]
This might not be the best way to do them, But its how I managed to get them working.

**Comment by Darkhax** [comment]
Let me sleep on it and I will get back to you on it when I wake up. I have a few issues with how unspecific the PR is (ignore file changes, build script changes and fixing deprecated code is all useful but unnecessary). I also have a few concerns with the formatting and documentation. I may just merge the PR and then make the specific changes since there are far too many formatting things to point out, and I have an automated script to format things.

File: src/test/java/net/darkhax/teslatest/tileentity/TileEntityAnalyzer.java [hunk 1]
```diff
@@ -28,7 +30,6 @@ public TileEntityAnalyzer() {
     
     @Override
     public void readFromNBT (NBTTagCompound compound) {
-
```
Comment (modmuss50) [inline src/test/java/net/darkhax/teslatest/tileentity/TileEntityAnalyzer.java:31]: Don't see a point to this change for this PR

File: src/test/resources/assets/teslatest/lang/en_US.lang [hunk 2]
```diff
@@ -1,4 +1,8 @@
-#Tile
-tile.teslatest.analyzer.name=Tesla Analyzer
-tile.teslatest.power.name=Creative Power Cell
-tile.teslatest.blackhole.name=Power Blackhole
+#Tile
+tile.teslatest.analyzer.name=Tesla Analyzer
+tile.teslatest.power.name=Creative Power Cell
+tile.teslatest.blackhole.name=Power Blackhole
+item.teslatest.battery.name=Battery
```
Comment (modmuss50) [inline src/test/resources/assets/teslatest/lang/en_US.lang:6]: Should have a #Item header

File: src/test/java/net/darkhax/teslatest/item/ItemBattery.java [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            return super.getMaxDamage(stack);
+    }
+
+    /*
+     * Sets item tooltip saying how much energy is stored and capacity
+     */
+    @Override
+    public void addInformation(ItemStack stack, EntityPlayer playerIn, List<String> tooltip, boolean advanced) {
+        super.addInformation(stack, playerIn, tooltip, advanced);
+        tooltip.add(I18n.format("label.teslatest.battery.normal",stack.getCapability(TeslaCapabilities.CAPABILITY_HOLDER,EnumFacing.DOWN).getStoredPower(),stack.getCapability(TeslaCapabilities.CAPABILITY_HOLDER,EnumFacing.DOWN).getCapacity()));
```
Comment (modmuss50) [inline src/test/java/net/darkhax/teslatest/item/ItemBattery.java]: Might be best to store the CAPABILITY_HOLDER would save this long line, and speed things up a tiny bit

**Comment by p455w0rd** [comment]
nice :)

**Pull request merged**